test/bundler_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class BundlerTest < Minitest::Test
+  def test_bundler?
+    assert Bootsnap.bundler?
```

Comment:
We prefer the `assert_predicate` method: `assert_predicate Bootsnap, :bundler?` because it gives a better error message.

Although we're running in a Bundler environment, I'd prefer not to rely on that to make this test work. We should explicitly set this up for success or failure